packages/sdk/utils/src/countries.ts

```diff
@@ -1,8 +1,7 @@
 import countryData from 'country-data'
-import { getExampleNumber } from './phoneNumbers'
-
 // more countries @ https://github.com/umpirsky/country-list
-const esData = require('../data/countries/es/country.json')
+import esData from './data/countries/es/country.json'
```

Comment:
nit [no need to fix]: would have been helpful to see this in a separate PR since it doesnt have to do with updating version #s, but rather fixing a bug